_data/gaming.yml

```
@@ -250,6 +250,13 @@ websites:
       email: Yes
       doc: https://robertsspaceindustries.com/faq/two-step-authentication
 
+    - name: Roblox
+      url: https://roblox.com
+      img: roblox.png
+      tfa: Yes
+      email: Yes
+      doc: https://blog.roblox.com/2016/09/roblox-introduces-2-step-verification/
```

I believe this help article better explains how to set 2FA up.
```suggestion
      doc: https://en.help.roblox.com/hc/en-us/articles/212459863
```